manifests/server/installdb.pp

```diff
@@ -14,12 +14,21 @@
 
     }
 
+    exec { 'remove-replication-params':
+      command => "sed -i '/server-id/ d' /usr/my.cnf; sed -i '/rpl_semi_sync_/ d' /usr/my.cnf",
```

Comment:
https://github.com/puppetlabs/puppetlabs-mysql/blob/master/manifests/params.pp has a variable for where the my.cnf file is located. Please use that instead of hardcoding a single location.